pendulum/pendulum.py

```diff
@@ -71,6 +71,10 @@ def _safe_create_datetime_zone(cls, obj):
             # pytz
             if hasattr(obj, 'localize'):
                 obj = obj.zone
+            elif hasattr(obj, 'utcoffset'):
+                delta = obj.utcoffset(None)
+
+                return FixedTimezone(delta.total_seconds() * 60 * 60)
```

Comment:
`FixedTimezone` expects an offset in seconds so it should be `FixedTimezone(delta.total_seconds())`